### PR TITLE
[Inji 434 & INJI 441]: Add events in receiver VC sharing flow and cancel button handle

### PR DESCRIPTION
**Pull request by Alka1703**
https://mosip.atlassian.net/browse/INJI-441
https://mosip.atlassian.net/browse/INJI-434
Handle cancel button on vc share flow on verifier side
Extract out Telemetry constants to a separate file

### CODE REVIEW

File: screens/Home/MyVcs/AddVcModal.tsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ import {OtpVerificationModal} from './OtpVerificationModal';
 import {IdInputModal} from './IdInputModal';
 import {useTranslation} from 'react-i18next';
 import {GET_INDIVIDUAL_ID} from '../../../shared/constants';
-import {TelemetryConstants} from '../../../shared/telemetry/TelemetryUtils';
+import {TelemetryConstants} from '../../../shared/telemetry/TelemetryConstants';
+import {Button, Column} from '../../../components/ui';
```
Comment (swatigoel) [inline screens/Home/MyVcs/AddVcModal.tsx:9]: only import, no usage?

File: machines/bleShare/request/requestMachine.ts [hunk 2]
```diff
@@ -241,11 +252,21 @@ export const requestMachine =
             },
             CONNECTED: {
               target: 'waitingForVc',
-              actions: ['setSenderInfo', 'setReceiverInfo'],
+              actions: [
+                'setSenderInfo',
+                'setReceiverInfo',
+                'sendVcSharingStartEvent',
```
Comment (swatigoel) [inline machines/bleShare/request/requestMachine.ts:258]: we should rename this to sendVcReceiveStartEvent.

File: machines/bleShare/request/requestMachine.ts [hunk 3]
```diff
@@ -241,11 +252,21 @@ export const requestMachine =
...
+              ],
             },
 
             DISCONNECT: {
               target: 'disconnected',
+              on: {
+                CANCEL: {
+                  target: '#request.cancelling',
+                  actions: ['sendVCSharingTerminatedEvent'],
```
Comment (swatigoel) [inline machines/bleShare/request/requestMachine.ts:267]: we should rename this to sendVCReceiveTerminatedEvent.

File: machines/bleShare/request/requestMachine.ts [hunk 4]
```diff
@@ -268,13 +289,15 @@ export const requestMachine =
               on: {
                 CANCEL: {
                   target: '#request.cancelling',
+                  actions: 'sendVCShareFlowTimeoutEndEvent',
```
Comment (swatigoel) [inline machines/bleShare/request/requestMachine.ts:292]: we should rename this to sendVCReceiveFlowTimeoutEndEvent.

File: machines/bleShare/request/requestMachine.ts [hunk 5]
```diff
@@ -268,13 +289,15 @@ export const requestMachine =
...
                 },
               },
             },
           },
           on: {
             DISCONNECT: {
               target: 'disconnected',
+              actions: ['sendVCSharingDisconnectedEvent'],
```
Comment (swatigoel) [inline machines/bleShare/request/requestMachine.ts:300]: we should rename this to sendVCReceiveDisconnectedEvent.

File: machines/bleShare/request/requestMachine.ts [hunk 6]
```diff
@@ -300,7 +323,10 @@ export const requestMachine =
                 },
                 FACE_INVALID: {
                   target: 'invalidIdentity',
-                  actions: 'setReceiveLogTypeUnverified',
+                  actions: [
+                    'setReceiveLogTypeUnverified',
+                    'sendVcShareFailedEvent',
```
Comment (swatigoel) [inline machines/bleShare/request/requestMachine.ts:328]: we should rename this to sendVcReceiveFailedEvent.

File: machines/bleShare/request/requestMachine.ts [hunk 7]
```diff
@@ -394,6 +420,7 @@ export const requestMachine =
                 'sendVcReceived',
                 'setReceiveLogTypeRegular',
                 'logReceived',
+                'sendVcShareSuccessEvent',
```
Comment (swatigoel) [inline machines/bleShare/request/requestMachine.ts:423]: we should rename this to sendVcReceiveSuccessEvent.

File: machines/bleShare/request/requestMachine.ts [hunk 8]
```diff
@@ -408,7 +435,11 @@ export const requestMachine =
               },
             },
             rejected: {
-              entry: ['setReceiveLogTypeDiscarded', 'logReceived'],
+              entry: [
+                'setReceiveLogTypeDiscarded',
+                'logReceived',
+                'sendVcRejectedEvent',
```
Comment (swatigoel) [inline machines/bleShare/request/requestMachine.ts:441]: we should rename this to sendVcReceiveRejectedEvent.

File: machines/bleShare/request/requestMachine.ts [hunk 9]
```diff
@@ -452,7 +483,11 @@ export const requestMachine =
             },
             savingFailed: {
               initial: 'idle',
-              entry: ['setReceiveLogTypeDiscarded', 'logReceived'],
+              entry: [
+                'setReceiveLogTypeDiscarded',
+                'logReceived',
+                'sendVcRejectedEvent',
```
Comment (swatigoel) [inline machines/bleShare/request/requestMachine.ts:489]: we should rename this to sendVcReceiveRejectedEvent.

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: machines/bleShare/request/requestMachine.ts (hunk 2 above)
Comment (Alka1703) [reply machines/bleShare/request/requestMachine.ts:258]: We have not Vc receive till now. It's just in connect state. Named it VCShare as the flow is VC share

File: screens/Home/MyVcs/AddVcModal.tsx (hunk 1 above)
Comment (Alka1703) [reply screens/Home/MyVcs/AddVcModal.tsx:9]: Removing it. Added by mistake

**Review by swatigoel — APPROVED** [review]

**Pull request merged**